src/lsp/cobol_ast/misc_descr.ml

```diff
@@ -721,21 +725,46 @@ let pp_arithmetic_mode ppf = function
   | StandardDecimal -> Fmt.pf ppf "STANDARD-DECIMAL"
 
 let pp_options_clause ppf = function
-  | Arithmetic am -> Fmt.pf ppf "ARITHMETIC %a" pp_arithmetic_mode am
+  | Arithmetic am ->
+      Fmt.pf ppf "ARITHMETIC %a" pp_arithmetic_mode am
   | DefaultRoundedMode rm ->
-    Fmt.pf ppf "DEFAULT ROUNDED MODE %a" pp_rounding_mode rm
+      Fmt.pf ppf "DEFAULT ROUNDED MODE %a" pp_rounding_mode rm
   | EntryConvention ec ->
-    Fmt.pf ppf "ENTRY-CONVENTION %a" pp_entry_convention ec
+      Fmt.pf ppf "ENTRY-CONVENTION %a" pp_entry_convention ec
   | FloatBinaryDefault em ->
-    Fmt.pf ppf "FLOAT-BINARY %a" Data_descr.pp_endianness_mode em
+      Fmt.pf ppf "FLOAT-BINARY %a" Data_descr.pp_endianness_mode em
   | FloatDecimalDefault ee ->
-    Fmt.pf ppf "FLOAT-DECIMAL %a" Data_descr.pp_encoding_endianness ee
+      Fmt.pf ppf "FLOAT-DECIMAL %a" Data_descr.pp_encoding_endianness ee
   | IntermediateRounding rm ->
-    Fmt.pf ppf "INTERMEDIATE-ROUNDING %a" pp_rounding_mode rm
+      Fmt.pf ppf "INTERMEDIATE-ROUNDING %a" pp_rounding_mode rm
 
 let pp_options_paragraph : options_paragraph Fmt.t =
-  Fmt.(
-    any "OPTIONS.@ " ++
-    box (list ~sep:sp (pp_with_loc pp_options_clause))
-    ++ any "."
-  )
+  Fmt.(any "OPTIONS.@ " ++
+       box (list ~sep:sp (pp_with_loc pp_options_clause)) ++
+       any ".")
+
+
+(* --- *)
+
+let pp_informational_paragraph_header ppf = function
```

Comment:
Functions with arity 2 should be defined with 2 arguments, `let f x = function ...` is bad for readability.

Reply:
I'd say the form with `function` reads much better for many specific cases, like pretty-printing procedures for instance (just need to visually match `pp_ … ppf` to spot those.